src/types/model/contentRelationship.ts

```diff
@@ -19,6 +18,5 @@ export interface CustomTypeModelContentRelationshipField<
 		select: typeof CustomTypeModelLinkSelectType.Document
 		customtypes?: readonly CustomTypeIDs[]
 		tags?: readonly Tags[]
-		text?: CustomTypeModelKeyTextField
```

Comment:
I think the idea is that content relationships are not links and thus don't need labels. What happens if someone wants to limit a link to only internal docs, not URLs? Wouldn't they need a Content Relationship?

Reply:
Hey, 

Yes it's the idea, to not have link allow for CR. The fact is, today they are really mixed. In SM UI you cannot select to only link to documents. So that why I did that.

You think we should be more flexible on that? 

At the end technically, if you edit manually your slice in SM it will work, custom types API will accept it. I was just thinking on prismic-client to not push for it as it's really name Content Relationship.

Reply:
Things might break if the `@prismicio/client` types are not 1:1 with what other tools allow. For example, incorrect types might be generated via `prismic-ts-codegen`, and users won't be able to access `myContentRelation.text` without TypeScript complaining.

> In SM UI you cannot select to only link to documents.

Today, the content relationship field is used for that purpose. It has to be done that way because the link field does not allow for limiting what is accepted. It's a workaround since, as you said, a link field would be more appropriate _if_ it had the capability of limiting what it accepts.

Since this is a new feature and there's no risk of making a breaking change, let's go with what the team already decided. We can always add `text` to the content relationship type if we find it is necessary. 👍